platform/aion/source/model.go

```diff
@@ -0,0 +1,18 @@
+package source
+
+type AionTransactionRPCResponse struct {
+	Content []AionTransaction 
+} 
+
+type AionTransaction struct {
```

Comment:
No need for `Aion` prefix. The prefix is the package name already (`source.Transaction` vs `model.Transaction`)